clang/test/PCH/cuda-kernel-call.cu

```diff
@@ -14,12 +16,19 @@ void kcall(void (*kp)()) {
 __global__ void kern() {
 }
 
+__host__ int overloaded_func();
```

Comment:
can you also add a comment here explaining what we're testing?

Reply:
Done